roles/StackStorm.st2/tasks/main.yml

```diff
@@ -100,3 +100,15 @@
     state: started
   loop: "{{ st2_services }}"
   tags: st2
+
+# https://github.com/ansible/ansible/issues/41313#issuecomment-520891625
```

Comment:
:+1:
nice trick.
Can We also include a small clarifying comment besides of URL

Reply:
done